src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -50,15 +50,49 @@ let uniqueId = 0;
 })
 export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDestroy {
 
+  /**
+   * Specifies the index of the repeater item to visually highlight as active.
+   * For example, use this property in conjunction with the
+   * [split view component](https://developer.blackbaud.com/skyux/components/split-view)
+   * to highlight a repeater item while users edit it. Only one item can be active at a time.
+   */
   @Input()
   public activeIndex: number;
 
+  /**
+   * Defines a string value to label the repeater list for accessibility.
+   * @default 'List of items'
+   */
   @Input()
   public ariaLabel: string;
 
+  /**
+   * Indicates whether users can change the order of items in the repeater list.
+   * Each repeater item also has `reorderable` property to indicate whether
+   * users can change its order.
+   */
   @Input()
   public reorderable: boolean = false;
 
+  /**
+   * Specifies a layout for the repeater list to indicate whether users can collapse
```

Comment:
In the existing docs, `expandMode` is marked as required. This doesn't make sense though since there is a default value. But just to make sure, removing the required tag is correct, right?

Reply:
Correct. It should not be marked as required.